x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeDataFrameAnalyticsIntegTestCase.java

```diff
@@ -233,6 +233,8 @@ protected static void assertThatAuditMessagesMatch(String configId, String... ex
         // Since calls to write the AbstractAuditor are sent and forgot (async) we could have returned from the start,
         // finished the job (as this is a very short analytics job), all without the audit being fully written.
         assertBusy(() -> assertTrue(indexExists(NotificationsIndex.NOTIFICATIONS_INDEX)));
+        client().admin().indices().prepareRefresh(".ml-notifications*").get();
```

Comment:
Is this needed taking into account that `fetchAllAuditMessages` also issues refresh request?

Reply:
Ah, good point. That comment got me confused. I'll remove.